cli/main.go

```diff
@@ -19,6 +19,8 @@ func main() {
 		}
 		path, err := exec.LookPath(fmt.Sprintf("linkerd-%s", args[0]))
 		if err == nil {
+			// The input arguments are templated.
```

Comment:
what do you mean by templated?

Reply:
Just referencing that the arguments here are passed in through a YAML file and that we should be okay to accept them since it's the user that decides the values: https://github.com/linkerd/linkerd2/blob/a268ff11c991ec0aa6c088479899ef065df7fe7b/cli/cmd/testdata/install_default.golden#L1873-L1883

Reply:
isn't this the CLI, not the controller?

Reply:
Oh whoops yep I'll change it.